webapp/channels/src/components/admin_console/filter/team_filter_dropdown/team_filter_checkbox.tsx

```diff
@@ -11,36 +8,33 @@ type Props = {
     updateOption: (checked: boolean, name: string) => void;
 };
 
-class TeamFilterCheckbox extends React.PureComponent<Props> {
-    toggleOption = () => {
-        const {checked, id, updateOption} = this.props;
+const TeamFilterCheckbox = ({
+    id,
+    name,
+    checked,
+    label,
+    updateOption,
+}: React.FC<Props>) => {
```

Comment:
The type here is wrong
```suggestion
}: Props) => {
```